thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/detector/email/filter/PrecisionRecallEvaluator.java

```diff
@@ -8,16 +8,13 @@
 import java.util.Collections;
 import java.util.List;
 import java.util.Properties;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
 
 /**
  * Utility class to evaluate the performance of a list of merged anomalies
  */
 public class PrecisionRecallEvaluator {
   private AlertFilter alertFilter;
-  private static final Logger LOG = LoggerFactory.getLogger(PrecisionRecallEvaluator.class);
 
   private int qualifiedTrueAnomaly; // Anomaly is labeled as true and is qualified
   private int qualifiedTrueAnomalyNotActionable; // Anomaly is labeled as TRUE_BUT_NOT_ACTIONABLE and is qualified
```

Comment:
@cyy0714 Hi Yungyu, can you change the logic here when converting non actionable to new trend after Neha's PR? This will effect the precision on anomaly report.